test/dlc_tests/sanity/test_safety_check.py

```diff
@@ -178,22 +172,27 @@ def test_safety(image):
         f"--entrypoint='/bin/bash' "
         f"{image}", hide=True)
     try:
-        run(f"{docker_exec_cmd} pip install safety yolk3k ", hide=True)
-        json_str_safety_result = safety_check.run_safety_check_on_container(docker_exec_cmd)
-        safety_result = json.loads(json_str_safety_result)
-        for vulnerability in safety_result:
-            package, affected_versions, curr_version, _, vulnerability_id = vulnerability[:5]
-            # Get the latest version of the package with vulnerability
-            latest_version = _get_latest_package_version(package)
-            # If the latest version of the package is also affected, ignore this vulnerability
-            if Version(latest_version) in SpecifierSet(affected_versions):
-                # Version(x) gives an object that can be easily compared with another version, or with a SpecifierSet.
-                # Comparing two versions as a string has some edge cases which require us to write more code.
-                # SpecifierSet(x) takes a version constraint, such as "<=4.5.6", ">1.2.3", or ">=1.2,<3.4.5", and
-                # gives an object that can be easily compared against a Version object.
-                # https://packaging.pypa.io/en/latest/specifiers/
-                ignore_str += f" -i {vulnerability_id}"
-        assert (safety_check.run_safety_check_with_ignore_list(docker_exec_cmd, ignore_str) == 0), \
-            f"Safety test failed for {image}"
+        SAFETY_FILE = "/opt/aws/dlc/info/safety_test_report.json"
+        safety_file_check = run(f"{docker_exec_cmd} test -e {SAFETY_FILE}", warn=True, hide=True)
```

Comment:
What happens when we want to run these on a canary basis, etc? How can we be sure we're checking for new vulnerabilities if this file exists?